backend/btrixcloud/operator/baseoperator.py

```diff
@@ -86,6 +99,10 @@ def compute_crawler_resources(self):
 
         p["crawler_cpu"] = crawler_cpu
         p["crawler_memory"] = crawler_memory
+        p["crawler_workers"] = num + 1
+        p["qa_cpu"] = qa_cpu
+        p["qa_memory"] = qa_memory
+        p["qa_workers"] = qa_num + 1
```

Comment:
I'm curious about the logic for `crawler workers = browser instances + 1`, that seems to be something new added in this PR? Is the idea that as we add more browser instances, they should also have more workers each? I wonder if this would be better set as its own field in the chart.

Reply:
It's slightly confusing logic, as it actually has a -1 above, because it is used for the 'extra' mem above base. But to get to the number of works, need to add 1. Maybe worth clearing this up, though.

Reply:
Right, I forgot about the -1. But we want the number of workers per browser instance to equal the total number of browser instances then, rather than being a separate value?

Reply:
We just compute a separate value for QA vs regular crawlers. Before we were just passing it in directly from helm chart, but now choosing the value based on whether its a QA run or not.